controllers/v1/profile.js

```diff
@@ -0,0 +1,55 @@
+const axios = require('axios')
+let profileHelper = require(MODULES_BASE_PATH + '/profile/helper')
+
+module.exports = class profile {
+	static get name() {
+		return 'profile'
+	}
+
+	async read(req) {
+		return new Promise(async (resolve, reject) => {
+			try {
+				// Define headers for the user service request
+				let headers = {
```

Comment:
@MallanagoudaB why we are defining headers in controller? calling external function from here is not accepted
controller always should pass values passed in api call (param, query. body) to helper function